runtime/src/main/java/io/quarkiverse/jasperreports/Constants.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkiverse.jasperreports;
+
+public interface Constants {
```

Comment:
Hmm don't love generic Constants class let me see what other extensions do

Reply:
Yeah, me either.  I would love to have a better pattern here.